nntrainer/tensor/bcq_tensor.cpp

```diff
@@ -247,7 +247,7 @@ void BCQTensor::save(std::ostream &file) {
   std::streamsize sz = static_cast<std::streamsize>(getMemoryBytes());
 
   NNTR_THROW_IF(sz < 0, std::invalid_argument)
-    << "save size: " << getMemoryBytes()
+    << "save size: " << bytes()
```

Comment:
I guess the save size should be getMemoryBytes
```suggestion
    << "save size: " << bytes()
```

Reply:
fixed it